packages/paste-core/components/spinner/src/index.tsx

```diff
@@ -45,4 +45,4 @@ Spinner.defaultProps = {
 };
 
 Spinner.displayName = 'Spinner';
-export {Spinner};
+export {Spinner, SpinnerProps};
```

Comment:
Could we change `SpinnerProps` to `Spinner` like we've done in other components like `Header` and `Paragraph`?

Reply:
I think we decided going forward we'd revert back to naming them `${Component}Props` just in case we do any components with props and state.